R/property.R

```diff
@@ -147,6 +173,43 @@ new_property <- function(class = class_any,
   out
 }
 
+check_prop_default <- function(default, class) {
```

Comment:
```suggestion
check_prop_default <- function(default, class, error_call = sys.call(-1))) {
```

And then pass explicitly to `stop.parent()`? That more closely matches the pattern we use in rlang so it's (a) battle tested and (b) easier for me to understand.